node/core/provisioner/src/metrics.rs

```diff
@@ -18,9 +18,13 @@ use polkadot_node_subsystem_util::metrics::{self, prometheus};
 
 #[derive(Clone)]
 struct MetricsInner {
+	/// Tracks successful/unsuccessful inherent data requests
 	inherent_data_requests: prometheus::CounterVec<prometheus::U64>,
+	/// How much time the `RequestInherentData` processing takes
 	request_inherent_data: prometheus::Histogram,
```

Comment:
```suggestion
	request_inherent_data_duration: prometheus::Histogram,
```

The doc comment is neat, it would be nice to mention duration in the name as well.

Reply:
Yes, good idea. But I'll rename the variable only. I won't touch the metric name so that I don't break any dashboard.